scipy/stats/tests/test_continuous.py

```diff
@@ -1031,19 +1031,25 @@ def test_shapes(self):
 
 
 class TestMakeDistribution:
-    @pytest.mark.parametrize('i, distdata', enumerate(distcont))
+    @pytest.mark.parametrize('i, distdata', enumerate(distcont + distdiscrete))
     def test_make_distribution(self, i, distdata):
         distname = distdata[0]
 
         slow = {'argus', 'exponpow', 'exponweib', 'genexpon', 'gompertz', 'halfgennorm',
                 'johnsonsb', 'kappa4', 'ksone', 'kstwo', 'kstwobign', 'powerlognorm',
-                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line'}
+                'powernorm', 'recipinvgauss', 'studentized_range', 'vonmises_line',
+                'betanbinom', 'zipf', 'logser', 'skellam'}  # discrete
         if not int(os.environ.get('SCIPY_XSLOW', '0')) and distname in slow:
             pytest.skip('Skipping as XSLOW')
 
-        if distname in {  # skip these distributions
-            'levy_stable',  # private methods seem to require >= 1d args
-            'vonmises',  # circular distribution; shouldn't work
+        if distname in {              # skip these distributions
+            'levy_stable',            # private methods seem to require >= 1d args
+            'vonmises',               # circular distribution; shouldn't work
+            'poisson_binom',          # vector shape parameter
+            'hypergeom',              # distribution functions need interpolation
+            'zipfian',                # distribution functions need interpolation
```

Comment:
Actually I can fix this one just by adding `k = np.floor(k)` to the CDF. We could also add that to the `_dispatch` method to make sure that no `_cdf_formula` is ever called with non-integral `k`. Not sure if we want to pull the trigger on that.